stan/math/prim/scal/prob/wiener_log.hpp

```diff
@@ -133,17 +134,25 @@ namespace stan {
       VectorView<const T_tau> tau_vec(tau);
       VectorView<const T_delta> delta_vec(delta);
 
+      size_t N_y_tau = max_size(y, tau);
+      for (size_t i = 0; i < N_y_tau; i++)
+        if (y_vec[i] <= tau_vec[i]) {
```

Comment:
No can do.  It doesn't print the right message out for vectors;  it assumes one of the value is an unnamed constant.  We could use a check_less_or_equal that's properly vectorized, because the way it is now involves a lot of ad-hoc checking and string wrangling.